src/ui.rs

```diff
@@ -171,6 +172,18 @@ impl Ui {
         Ok(())
     }
 
+    pub fn write_clap_err_and_exit(&mut self, err: clap::Error) -> ! {
+        // Color parsing must never panic, otherwise we'll never print help.
+        let color = use_color(color_setting(&self.settings));
+        let clap_str = if color {
+            err.render().ansi().to_string()
+        } else {
+            err.render().to_string()
+        };
+        self.write(clap_str.as_str()).ok();
```

Comment:
Is this `as_str` necessary?

Reply:
Unless I'm missing some Rust magic, I think so, since`.to_string()` returns `String` but `ui.write()` accepts `&str`

Reply:
`String` implemenets `Deref<Target=str>`, which means that `&String` coerces to `&str`.

Reply:
So this means that I should write `&clap_str` instead?

Reply:
Yes